charts/incubator/flashpaper/values.yaml

```diff
@@ -0,0 +1,51 @@
+image:
+  pullPolicy: IfNotPresent
+  repository: ghcr.io/andrewpaglusch/flashpaper
+  tag: latest@sha256:91e23442eec448d0ceea4e71a277fb204d22d5ed0980570782403aa7c0e8400e
+persistence:
+  data:
+    enabled: true
+    mountPath: /var/www/html/data
+  varrun:
+    enabled: false
+
+portal:
+  open:
+    enabled: true
+securityContext:
+  container:
+    readOnlyRootFilesystem: false
+    runAsGroup: 0
+    runAsNonRoot: false
+    runAsUser: 0
```

Comment:
Gotta love when a "security" focused software runs as root even when there is zero reasons too :D

Reply:
I don't think non-root user was attempted TBH